WISE-Community/WISE-Client
# feat(Peer Chat): Send a notification when a Peer Chat message is sent

## Changes
- When a Peer Chat message is sent, a notification is also sent to Peer Chat members

## Test
- Make sure a notification shows up when you are not on the Peer Chat step
- Make sure a notification does not show up when you are on the Peer Chat step
- Make sure only one notification shows up even if multiple messages are sent
- This scenario
  - Student is on a Peer Chat step
  - Student receives a notification for the Peer Chat step but we don't show it to them because they are on the same step as the notification
  - Refresh the VLE
  - The notification should still not show up

Closes #509
